fastlane_core/lib/fastlane_core/ipa_file_analyser.rb

```diff
@@ -19,6 +19,7 @@ def self.fetch_app_version(path)
 
     # Fetches the app platform from the given ipa file.
     def self.fetch_app_platform(path)
+      path = File.expand_path(path)
```

Comment:
Should this be happening in the methods above too? Or maybe at the call site for all of these methods?